src/twisted/logger/__init__.py

```diff
@@ -85,6 +85,9 @@ def oops(self, data):
     # From ._json
     "eventAsJSON", "eventFromJSON",
     "jsonFileLogObserver", "eventsFromJSONLogFile",
+
+    # From _capture
```

Comment:
Consistency nitpick: all the other "From"s are `From ._whatever` and this one skips the dot :)

Reply:
GAWD I can't believe I did that